similarity/src/main/java/edu/illinois/cs/cogcomp/sim/LevensteinSim.java

```diff
@@ -0,0 +1,24 @@
+package edu.illinois.cs.cogcomp.sim;
```

Comment:
todo: try replacing the levenstein scorer with that from core-utilities: https://github.com/CogComp/cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/algorithms/LevensteinDistance.java